src/query_results.js

```diff
@@ -356,35 +356,35 @@ module.exports = class QueryResult {
       //     "n0-NCBIGene:3630_&_n1-MONDO:0005068_&_n2-PUBCHEM.COMPOUND:43815"
       //     "n0-NCBIGene:3630_&_n1-MONDO:0005010_&_n2-PUBCHEM.COMPOUND:43815"
 
-      if (!preresultsByUniqueResultID.hasOwnProperty(uniqueResultID)) {
-        preresultsByUniqueResultID[uniqueResultID] = [];
+      if (!ucResultsByResultID.hasOwnProperty(uniqueResultID)) {
+        ucResultsByResultID[uniqueResultID] = [];
       }
-      preresultsByUniqueResultID[uniqueResultID].push(preresult)
+      ucResultsByResultID[uniqueResultID].push(ucResult)
     });
 
-    const consolidatedPreresults = toPairs(preresultsByUniqueResultID).map(([uniqueResultID, preresults]) => {
-      debug(`result ID: ${uniqueResultID} has ${preresults.length}`)
+    const consolidatedResults = toPairs(ucResultsByResultID).map(([uniqueResultID, ucResults]) => {
+      debug(`result ID: ${uniqueResultID} has ${ucResults.length}`)
       // spread is like Fn.apply
       // TODO: maybe just use ...
-      return spread(zip)(preresults).map(preresultRecords => {
-        const preresultRecord0 = preresultRecords[0];
+      return spread(zip)(ucResults).map(ucResultRecords => {
+        const ucResultRecord_0 = ucResultRecords[0];
         const consolidatedPreresultRecord = {
```

Comment:
Appears the intent is to get rid of the term `preresult`, so `consolidatedPreresultRecord` could be changed to something else. I used the term `preresult` to indicate a data structure that is used internally here before we produce the TRAPI result.

Reply:
`consolidatedPreresultRecord` can be thought of as all the valid data corresponding to one hop of the query graph for one TRAPI result.

Reply:
Hmm, I must have missed this. Would it be accurate to call this a `consolidatedResult`? I'm a little fuzzy on what's going on with the data structures in this part of the function..

Reply:
I agree -- this section is somewhat confusing. I'm not sure there's an obvious way to make it more clear, because it's tightly coupled to the actual underlying data processing, which is itself complicated.

The main idea is that a record is associated with just one hop, while a result must have at least one record for every hop in the query graph. I don't think `consolidatedResult` would be right, because IIRC, `consolidatedPreresultRecord` is just the consolidated data associated with one or more records for a single hop. Maybe `consolidatedRecord` would be appropriate?

Reply:
just pasting in some context here: I get the sense that this work https://github.com/biothings/bte_trapi_query_graph_handler/pull/88#issuecomment-1026413433 from @ariutta was not used by @tokebe when he went through the whole package....do we need to discuss differences between this PR and that previous work - and reach agreement on how to proceed?